client/my-sites/plans/current-plan/current-plan-thank-you/style.scss

```diff
@@ -27,27 +27,18 @@
 	}
 
 	.button {
-		background-color: var( --color-primary );
-		border-color: var( --color-primary-dark );
 		padding-right: 80px;
 		padding-left: 80px;
 
 		@include breakpoint( '<660px' ) {
 			text-align: center;
 			width: 100%;
 		}
-
-		&:hover,
-		&:focus {
-			background-color: var( --color-primary-400 );
-			border-color: var( --color-primary-dark );
-		}
 	}
 }
 
 .current-plan-thank-you__illustration {
 	margin-right: 32px;
-	width: 35%;
```

Comment:
Not entirely sure why this was needed. @Aurorum ?

Reply:
The intention was to make the image a bit smaller on smaller screens.

**Before:**

<img width="649" alt="Screenshot 2019-09-05 at 16 46 52" src="https://user-images.githubusercontent.com/43215253/64357646-f6c00b80-cffc-11e9-8e6c-80442ad985b7.png">

**This PR:**
(I felt the image was a bit too wide)
<img width="644" alt="Screenshot 2019-09-05 at 16 46 47" src="https://user-images.githubusercontent.com/43215253/64357570-d1cb9880-cffc-11e9-81cb-bbfae7bd9988.png">

I didn't realise that would minimise the image so much on Firefox though! :)

Reply:
Ahhh, makes sense! @jeffgolenski thoughts if this is ok as-is or needs tweaking for medium size screens? :-)